addons/common/functions/fnc_doAnimation.sqf

```diff
@@ -62,8 +64,19 @@ switch (_priority) do {
         };
     };
     case 2 : {
-        // Execute on all machines. SwitchMove has local effects.
-        [_unit, format ["{_this switchMove '%1'}", _animation]] call FUNC(execRemoteFnc);
+        // try playMoveNow first
+        if (_unit == vehicle _unit) then {
+            [_unit, format ["{_this playMoveNow '%1'}", _animation], _unit] call FUNC(execRemoteFnc);
```

Comment:
Earlier in this function it states that `playMoveNow` only needs to be broadcasted by to all clients when not in a veihcle.